client/post-editor/edit-post-status/index.jsx

```diff
@@ -187,6 +188,7 @@ export class EditPostStatus extends Component {
 				<Revisions
 					revisions={ this.props.post && this.props.post.revisions }
 					adminUrl={ adminUrl }
+					toggleNestedSidebar={ this.props.toggleNestedSidebar }
```

Comment:
This looks a bit opaque to me. Why do we need to pass as a prop? What does `nestedSidebar` mean? Is this only used for revisions? Can it be `toggleRevisions`?